engine/src/test/java/org/pentaho/di/trans/steps/tableoutput/TableOutputTest.java

```diff
@@ -202,4 +220,77 @@ public void testInit_unsupportedConnection() {
     KettleException ke = new KettleException( unsupportedTableOutputMessage );
     verify( tableOutputSpy, times( 1 ) ).logError( "An error occurred intialising this step: " + ke.getMessage() );
   }
+
+  @Test
+  public void validationRowMetaTest() throws Exception {
+    Method m = TableOutput.class.getDeclaredMethod( "isValidRowMeta", RowMetaInterface.class );
```

Comment:
If you really need to directly call a method that would otherwise be private, the usual way is to make it accessible and add a VisibleForTesting annotation. Using reflection adds needless complexity and makes tracking usage of that method very difficult.

Reply:
Agreed, but this is a moved method. Added comment above for the same.